tests/handler/test_handler_validates_service_handler_collection.py

```diff
@@ -18,11 +18,26 @@ class Service:
 
 
 def test_services_are_collected():
+    class OpHandler(nexusrpc.handler.SyncOperationHandler[int, int]):
+        async def start(
+            self,
+            _: nexusrpc.handler.StartOperationContext,
+            _input: int,
+        ) -> int:
+            return Output(value=1)  # type: ignore
+
+        async def cancel(
+            self,
+            _: nexusrpc.handler.StartOperationContext,
+            _token: str,
+        ) -> int:
+            return Output(value=1)  # type: ignore
```

Comment:
I think these can return the expected types, right? I'll just merge into the feature branch and update the tests in the feature branch.